vendor/wp-now/src/wp-now.ts

```diff
@@ -563,3 +572,15 @@ export async function moveDatabasesInSitu( projectPath: string ) {
 		fs.rmSync( wpContentPath, { recursive: true, force: true } );
 	}
 }
+
+function addDefaultUmaskPlugin( php: PHP, documentRoot: string ) {
+	php.writeFile(
+		'/internal/shared/mu-plugins/override-default-umask.php',
+		`<?php
+function set_default_umask()
+{
+    umask(0022);
+}
+add_action('init', 'set_default_umask', -9999);`
```

Comment:
The plugin is executed with the higher priority to ensure the `umask` is set before other plugins.